src/org/sosy_lab/llvm_j/Value.java

```diff
@@ -1403,6 +1410,18 @@ public long constIntGetSExtValue() {
     return LLVMLibrary.LLVMConstIntGetSExtValue(value);
   }
 
+  /** Returns the zero extended value for an integer constant value. */
+  public double constRealGetDouble() {
+    LLVMLibrary.LLVMBool precisionLoss = new LLVMLibrary.LLVMBool();
+    double ret =
+        LLVMLibrary.LLVMConstRealGetDouble(value, precisionLoss);
+
+    if (Utils.llvmBoolToJavaBool(precisionLoss))

```

Comment:
To fix the Checkstyle failure:
```suggestion
    if (Utils.llvmBoolToJavaBool(precisionLoss)) {
```